### PR TITLE
Fix some linting rules in codebase

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: packages/components/src/Components/Battery/Batery.test.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ describe('Battery component', () => {
         it('NullBatery, default', () => {
             const wrapper = shallow(<Battery severity={''} label={''}/>);
             expect(toJson(wrapper)).toMatchSnapshot();
+            // eslint-disable-next-line no-console
```
Comment (Hyperkid123) [inline packages/components/src/Components/Battery/Batery.test.js]: I think that we should permit warn and error but blacklist log.

We could use this rule globally: `no-console: ["error", { allow: ["warn", "error"] }]`

What do you think?

File: packages/components/src/Components/Breadcrumbs/Breadcrumbs.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import classnames from 'classnames';
 import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
 
 const Breadcrumbs = ({ items, current, className, onNavigate, ...props }) => {
+    // eslint-disable-next-line no-console
```
Comment (Hyperkid123) [inline packages/components/src/Components/Breadcrumbs/Breadcrumbs.js]: ^^

File: packages/components/src/Components/Main/Main.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ export class Main extends Component {
         };
     }
     render () {
+        // eslint-disable-next-line no-unused-vars
```
Comment (Hyperkid123) [inline packages/components/src/Components/Main/Main.js]: You can also use this rule to permit spread operator siblings
https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

**Review by Hyperkid123 — COMMENTED** [review]
I think avoid some of those `/* eslint-...` by adding/editing of certain rules.

File: packages/components/src/Components/Battery/Batery.test.js (hunk 1 above)
Comment (karelhala) [reply packages/components/src/Components/Battery/Batery.test.js]: I don't really like `error` for logs, but allowing warnings and errors is a good idea. I'll update it!

**Review by Hyperkid123 — APPROVED** [review]
:name_badge: :  :name_badge:  :name_badge:  :name_badge:  :snake:

**Pull request merged**